dashboards-observability/public/components/custom_panels/helpers/utils.tsx

```diff
@@ -299,7 +299,7 @@ export const displayVisualization = (metaData: any, data: any, type: string) =>
         rawVizData: data,
         query: { rawQuery: metaData.query },
         indexFields: {},
-        userConfigs: data?.jsonData,
+        userConfigs: {},
```

Comment:
Why are we change it from metaData.user_configs to data?.jsonData and removed it in the end? Did we do BWC?

Reply:
+1 @SivaprasadAluri Can you please explain why was this change added?

Reply:
@mengweieric,@ps48 sorry i had reverted back to metaData.user_configs but missed the change during the last commit, its been reverted and tested.